extensions/hp-extension/src/setViewportZoomPan.ts

```diff
@@ -0,0 +1,8 @@
+export default (id,value,viewport) => {
+  const {zoom,pan, flipHorizontal, flipVertical} = value;
```

Comment:
if we default zoom to 1 (ie. {zoom =1,...) we can reduce following if to just if(zoom!=1)